test_conformance/events/test_event_dependencies.cpp

```diff
@@ -275,6 +277,10 @@ int test_event_enqueue_wait_for_events_run_test(
 
     for (loop_count = 0; loop_count < TEST_COUNT; loop_count++)
     {
```

Comment:
think we can move the definition of `int event_count` from being at the function scope to being defined as `int event_count = 0;` at the start of this loop. So we also wouldn't need `event_count = 0;` at the end of the loop.

Reply:
done